pkg/apiserver/diagnose/.gitignore

```diff
@@ -0,0 +1 @@
+/embedded_diagnose_report_template.go
```

Comment:
I prefer to embed the generated template source code into the repository, since Dashboard will also be included in the PD (see https://github.com/pingcap-incubator/tidb-dashboard#getting-started-pd-with-tidb-dashboard ). Embedding source code in the repository now can make it easy.